cloudify_netconf/xml_rpc.py

```diff
@@ -182,7 +183,7 @@ def _parse_response(ctx, xmlns, netconf_namespace, response,
                 'Unexpected key in response: {response}'.format(
                     response=filters.shorted_text(xml_dict)))
         reply = \
-            [v for k, v in xml_dict.viewitems()
+            [v for k, v in list(xml_dict.items())
```

Comment:
Since you are not deleting any items `list()` shouldn't be necessary. Why are you using it?

Also, you can do `[v for _, v in ...]`.